packages/www/docs/guides/start-live-streaming/webhook.mdx

```diff
@@ -102,6 +102,40 @@ this account.
       description:
         "This fires when we are no longer sending video to the multistream target.",
     },
+    {
```

Comment:
I will update the "new" webhook page with this information:https://github.com/livepeer/livepeer-studio-docs/blob/main/docs/guides/livestreaming/webhook.mdx